imports/plugins/core/orders/server/no-meteor/resolvers/Order/orderStatus.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @name "Order.orderStatus"
+ * @method
+ * @memberof Order/GraphQL
+ * @summary Displays a human readable status of order state
+ * @param {Object} order - Result of the parent resolver, which is a Order object in GraphQL schema format
+ * @return {String} A string of the order status
+ */
+export default async function orderStatus(order) {
+  const { workflow: { status } } = order;
+
```

Comment:
@aldeed what are your thoughts on different labels / workflow displays here?

What this does is take our `workflow.status` data, and xform it into a more human readable label to go into the Orders account page. We (@rymorgan @cassytaylor and I) were thinking that "new" doesn't really mean much to anyone, so we'd go directly with "Order Received". "processing" could also be iffy to an end user, so we'll keep it "Order Received at that point".

Reply:
@kieckhafer I'm all for readable labels but we probably should not do it with hard-coded values. Ideally these would be customizable and language-specific. The `label` resolver could take a `language` arg as we've done elsewhere, and values could come from a language-to-label map somewhere in shop settings?

```js
orderStatusLabels: {
  new: [
    { language: "en", label: "Order received" }
  ]
}
```

Otherwise it could be left up to frontend code, but that's not as nice since there could be inconsistencies in what label each frontend uses.